database/seed.js

```diff
@@ -0,0 +1,44 @@
+const db = require("./index.js");
+const Listing = require("./Listing.js");
+
+const listings = [
+  {
+    documentId: 1,
+    listingId: 1,
```

Comment:
I think  Justin is using a starting id number for his listings 1001 and up. If you add listing id 1 - no listing will be found